lib/galaxy/tool_util/toolbox/views/definitions.py

```diff
@@ -77,8 +77,11 @@ class ItemsFrom(BaseModel):
 ]
 
 
-class HasItems:
-    items: Optional[List['RootContent']]
+C = TypeVar('C', bound="RootContent")
```

Comment:
Would this be an alternative? https://pydantic-docs.helpmanual.io/usage/postponed_annotations/#self-referencing-models
I guess that's what the error message was hinting at ?

Reply:
I had a version that used that but Section allowing all RootContent allows Sections and this broke a unit test and is just wrong.

This seemed to be better on Python 3.8... but I guess it fails under Python 3.6... hmm... sorry if this is slowing down the release.

Reply:
No worries about the release, we moved the freeze to Thursday anyway, lot of open PRs at this point.

Reply:
xref https://github.com/samuelcolvin/pydantic/issues/1547#issuecomment-636466257

Well at least I'm certain "the correct" solution won't work so I can pick a hack and feel okay.